fcl/gen/numi/genie_icarus_numioffaxis.fcl

```diff
@@ -90,6 +90,9 @@ icarus_genie_NuMI: {
 icarus_genie_NuMI_rock: @local::icarus_genie_NuMI
 icarus_genie_NuMI_rock.DetectorLocation: "icarus-numi-rock"
 
+# RHC Config
+icarus_genie_NuMI_RHC: @local::icarus_genie_NuMI
+icarus_genie_NuMI_RHC.FluxSearchPaths: "/cvmfs/sbn.osgstorage.org/pnfs/fnal.gov/usr/sbn/persistent/stash/physics/beam/GENIE/NuMI/standard/v02_00_RHC/"
```

Comment:
Following our discussion on the numi-nus channel, I confirm this is the location of the dk2nu files produced for NOvA 3rd, analysis, original location: /pnfs/numix/flux/g4numi_syst_3rd_ana/test_g4numi_v6/me000z-200i/minervame/run0015